appengine/standard/firebase/firetactoe/app.yaml

```diff
@@ -0,0 +1,11 @@
+runtime: python27

```

Comment:
(Github won't let me comment on a binary file): Please remove .DS_Store. I'd recommend adding it to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).